scrapy/item.py

```diff
@@ -56,6 +74,12 @@ class DictItem(MutableMapping, BaseItem):
 
     fields = {}
 
+    def __new__(cls, *args, **kwargs):
+        if cls is DictItem:
+            warn('scrapy.item.DictItem is deprecated, please use '
+                 'scrapy.item.Item instead', ScrapyDeprecationWarning)
```

Comment:
how does a warning message look like, does it point to user code? I wonder if a non-default stacklevel can make the message more clear.